npm/cypress-schematic/src/schematics/ng-generate/cypress-test/index.ts

```diff
@@ -53,14 +53,18 @@ export default function (options: Schema): Rule {
     }
 
     if (options.path === undefined) {
-      options.path = `${project.root}/cypress/e2e`
+      options.path = options.testingType === 'component' ? `${project.sourceRoot}/${project.prefix}` : `${project.root}/cypress/e2e`
     }
 
-    const templateSource = apply(url('../files/__path__'), [
+    console.log(`Creating new ${options.testingType} spec named: ${options.name}`)
+
+    const templatePath = options.testingType === 'component' ? '../files/ct/__path__' : '../files/e2e/__path__'
+    const templateSource = apply(url(templatePath), [
       applyTemplates({
         classify: strings.classify,
         dasherize: strings.dasherize,
-        name: options.name,
+        name: options.testingType === 'component' ? `${options.name.toUpperCase()}Component` : options.name,
```

Comment:
`toUpperCase` results in:

```ts
import { MYSPECComponent } from './MySpec.component'

describe('MYSPECComponent', () => {
  it('should mount', () => {
    cy.mount(MYSPECComponent)
  })
})
```

We are using classify in the template, so we should be able to remove the uppercase transform

Reply:
This has been removed. Good catch!